.travis.yml

```diff
@@ -1,8 +1,13 @@
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
 language: node_js
 node_js:
   - "6"
```

Comment:
We don't support Node < 8.2.1 as far as I remember

Reply:
I will remove Node.js 6
package.json says "node": ">=7.6.0"
if this is not true anymore, it should get updated as well.

Reply:
Our build page is saying Node.js >= 6.9.1 as well
 
https://github.com/inexorgame/inexor-core/wiki/Build